### PR TITLE
Added ssl authentation to the nats extensions

### PR DESCRIPTION
**Pull request by aldettinger**
[ ] An issue should be filed for the change unless this is a trivial change (fixing a typo or similar). One issue should ideally be fixed by not more than one commit and the other way round, each commit should fix just one issue, without pulling in other changes.
[ ] Each commit in the pull request should have a meaningful and properly spelled subject line and body. Copying the title of the associated issue is typically enough. Please include the issue number in the commit message prefixed by #.
[ ] The pull request description should explain what the pull request does, how, and why. If the info is available in the associated issue or some other external document, a link is enough.
[ ] Phrases like Fix #<issueNumber> or Fixes #<issueNumber> will auto-close the named issue upon merging the pull request. Using them is typically a good idea.
[ ] Please run mvn process-resources -Pformat (and amend the changes if necessary) before sending the pull request.
[ ] Contributor guide is your good friend: https://camel.apache.org/camel-quarkus/latest/contributor-guide.html

### CODE REVIEW

File: integration-tests/nats/src/test/java/org/apache/camel/quarkus/component/nats/it/NatsTestResource.java [hunk 1]
```diff
@@ -63,6 +65,20 @@
         Integer noAuthPort = noAuthContainer.getMappedPort(NATS_SERVER_PORT);
         String noAuthBrokerUrl = String.format("%s:%s", noAuthIp, noAuthPort);
 
+        // Start the container needed for the TLS authentication test
+        tlsAuthContainer = new GenericContainer(NATS_IMAGE).withExposedPorts(NATS_SERVER_PORT)
+                .withClasspathResourceMapping("certs", "/certs", BindMode.READ_ONLY)
```
Comment (aldettinger) [inline integration-tests/nats/src/test/java/org/apache/camel/quarkus/component/nats/it/NatsTestResource.java:70]: It uses `ClassLoader.getResource(...)` under the hood, so I think the test could be executed on platform too.

**Comment by aldettinger** [comment]
Better merge it after 1.1.0 release.

**Review by ppalaga — APPROVED** [review]

**Comment by ppalaga** [comment]
> Better merge it after 1.1.0 release.

Why? I do not see anything risky here (maybe I am missing something?)

**Review by oscerd — APPROVED** [review]

**Comment by aldettinger** [comment]
Ok, let's merge now then. Thanks for review guys.

**Pull request merged**